tools/clang/lib/SPIRV/TypeTranslator.cpp

```diff
@@ -766,7 +765,7 @@ TypeTranslator::translateSampledTypeToImageFormat(QualType sampledType) {
       }
     }
   }
-  emitError("unimplemented resource result type was used");
+  emitError("resource type %0 unimplemented") << sampledType.getAsString();
```

Comment:
Can actually directly `<< sampledType` here?

Reply:
Not sure... Will there be implicit casting to string? Can address in followup CL